css-flexbox/align-items.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      .container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+      .card-wrapper {
+        flex-basis: calc(100% / 3);
+      }
+      #card-deck {
+        min-height: 100vh;
+      }
```

Comment:
Because there was already a `card-deck` `id` on the `div`, your `.container` selector and `#card-deck` selector declaration blocks could have been combined into one CSS ruleset.
```
#card-deck {
      min-height: 100vh;
      display: flex;
      align-items: center;
      justify-content: center;
    }
```